core/domain/subtopic_page_domain.py

```diff
@@ -347,18 +421,21 @@ def update_page_contents_audio(self, new_page_contents_audio):
         self.page_contents.recorded_voiceovers = new_page_contents_audio
 
     def update_page_contents_written_translations(
-            self, new_page_written_translations_dict):
+            self,
+            new_page_written_translations_dict:
+            Dict[str, Dict[str, Dict[str, Dict[str, object]]]]
```

Comment:
Why `object`?

Reply:
When I ran the _test.py file, I was prompted that Dict[str, Dict[str, Dict[str, Dict[str, object]]]] was the expected type. Hence, I used the same.